### PR TITLE
Add dbt build command as an operator

### PR DESCRIPTION
**Pull request by annabaringer**
Hi y'all! Thanks for maintaining this repo! 

This PR adds an operator for dbt's build command. 

cc: @jackwelty from my team

### CODE REVIEW

File: README.md
Comment (emily-flambe) [inline README.md:66]: 🎉

**Review by emily-flambe — APPROVED** [review]
Sick!! Looks good to me, assuming you've tested it and etc. you have my blessing to merge this in! 🇺🇸 🇺🇸 🇺🇸

**Comment by emily-flambe** [comment]
qq @annabaringer and @jackwelty, are y'all using the pypi distribution of this? If so then we can make a new release after this is merged in so that you can use this Cool New Functionality to achieve your goals ✨

**Review by jitoquinto — CHANGES_REQUESTED** [review]
Sweet! Thank you for contributing! One ask though, could you bump the version number on the `airflow_dbt_cta/__version__.py` file? It would just be like [this commit](https://github.com/community-tech-alliance/airflow-dbt/commit/daceb11553f2ba98a3ed43c2579ad039bdc235d2)!

**Review by jitoquinto — APPROVED** [review]
Awesome thank you!! 🚀

**Comment by jitoquinto** [comment]
> > qq @annabaringer and @jackwelty, are y'all using the pypi distribution of this? If so then we can make a new release after this is merged in so that you can use this Cool New Functionality to achieve your goals ✨
> 
> Yes we are!! That would be amazing, thank you! FYI I don't think I have permission to merge, but we are happy when you are!

Okay cool! I'll go ahead and merge + push a new version up to pypi!

**Comment by jitoquinto** [comment]
@annabaringer this is now up on pypi! - https://pypi.org/manage/project/airflow-dbt-cta/release/0.0.9/

Thank you again!!

**Comment by jackwelty** [comment]
Thank you so much @jitoquinto and @emily-flambe !!

**Pull request merged**
